tests/macros/utils.js

```diff
@@ -183,20 +341,55 @@ function lintHTML(html, fragment = true) {
         });
         return null;
     } catch (error) {
-        const error_message = error.message
+        const error_message = /** @type {Error} */ (error).message
             .split(os.EOL)
             .filter(line => line.startsWith('Error: '))
             .join(os.EOL);
         return error_message;
     }
 }
 
+/**
+ * Reads a generic fixture file from the `fixtures` directory.
+ *
+ * @param {string|string[]} filePath
+ *        A path to a file relative to the `fixtures` directory.
+ * @param {string|{encoding?:string,flag?:string}} [options]
```

Comment:
I'd recommend just making this a string. I can't think of any case where readFileSync() would need anything other than the default "r" for the flag, can you?  If you're going to keep the option of using an object, maybe just refer tot he node docs in the comment. Because as it stands now you never actually say what the flag is for.

Reply:
I copied the description for this from the node docs.